frontend/src/metabase/routes.jsx

```diff
@@ -154,6 +155,10 @@ export const getRoutes = store => {
             <ModalRoute path="move" modal={MoveCollectionModal} noWrap />
             <ModalRoute path="archive" modal={ArchiveCollectionModal} />
             <ModalRoute path="permissions" modal={CollectionPermissionsModal} />
+            <ModalRoute
+              path="cleanup-questions"
```

Comment:
@luizarakaki let me know if you'd prefer another url (e.g. `/collection/2-examples/cleanup-questions`)

Reply:
is `/move-questions-dashboard` too long?

I think it is better just to differentiate from `/collection/2/cleanup`